tests/test_modeling_bert.py

```diff
@@ -477,3 +483,45 @@ def test_model_from_pretrained(self):
         for model_name in BERT_PRETRAINED_MODEL_ARCHIVE_LIST[:1]:
             model = BertModel.from_pretrained(model_name)
             self.assertIsNotNone(model)
+
+
+@require_sentencepiece
+@require_tokenizers
```

Comment:
The tests do not require sentencepiece nor tokenizers as far as I can see.

Reply:
Good catch. Fixed for both `test_modeling_bert.py` and `test_modeling_roberta.py`.